src/github/folderRepositoryManager.ts

```diff
@@ -744,6 +747,44 @@ export class FolderRepositoryManager implements vscode.Disposable {
 		return undefined;
 	}
 
+	private async getTeamReviewersFromGlobalState(): Promise<{ [key: string]: ITeam[] } | undefined> {
+		Logger.appendLine('Trying to use globalState for team reviewers.');
+
+		const teamReviewersCacheLocation = vscode.Uri.joinPath(this.context.globalStorageUri, 'teamReviewers');
+		let teamReviewersCacheExists;
+		try {
+			teamReviewersCacheExists = await vscode.workspace.fs.stat(teamReviewersCacheLocation);
+		} catch (e) {
+			// file doesn't exit
+		}
+		if (!teamReviewersCacheExists) {
+			return undefined;
+		}
+
+		const cache: { [key: string]: ITeam[] } = {};
+		const hasAllRepos = (await Promise.all(this._githubRepositories.map(async (repo) => {
+			const key = `${repo.remote.owner}/${repo.remote.repositoryName}.json`;
```

Comment:
There could be name collisions with enterprise instances.

Reply:
I think we could only get a collision if there are two unrelated enterprises with the same owner + repo name combination. Is that correct?

Reply:
Because the cache key makes no distinction between repo origins, yes, two unrelated enterprises might collide. But more importantly and likely would be a collision between `github.com` repos and any enterprise.

Example:
* `github.com/microsoft/vscode`
* `github.contoso.net/microsoft/vscode`

While it may be unlikely that these examples exist and represent distinct teams, they are indeed two different sources of truth for information when querying/caching on the context `microsoft/vscode`

Reply:
Got it, thanks. I'll update the key in another PR.